todoapp/todos/utils.py

```
@@ -54,8 +69,11 @@ def fetch_all_todo_list_with_user_details():
     Util to fetch given user's tod list
     :return: list of dicts - List of todos
     """
-    # Write your code here
-    pass
+    todos = todos_models.Todo.objects.select_related("user").all()
+    serializer = todos_serializers.TodoSerializer(
+        todos, many=True, fields=("id", "name", "status", "created_at", "creator")
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)